app/models/public_key.py

```diff
@@ -0,0 +1,9 @@
+from sqlmodel import SQLModel
+
+
+class PublicKeyModel(SQLModel):
+    key: str
+
+    @classmethod
+    def from_str(cls, key: str) -> "PublicKeyModel":
+        return cls(key=key)
```

Comment:
Correcciones:
- En ningun micro-servicio, hemos agregado el sufijo "Model" a los modelos.
- En lugar de instanciar al PublicKeyModel usando el metodo `from_str` me parece más natural crear un constructor __init__(self) ó directamente instanciarlo como`PublicKeyModel(key=<key>)`
```suggestion
class PublicKey(SQLModel):
    key: str

    def __init__(self, key: str) -> None:
        self.key = key
```

Reply:
Se remplazo el nombre de la clase y como depende SQLModel no se puede hacer un init sin realizar un super().init() lo cual complejiza el codigo ya que el super().__init__() requiere parametros que no tengo.